commitlint.config.ts

```diff
@@ -81,6 +81,7 @@ module.exports = {
         'trailing-whitespace': [RuleStatus.Error, 'always'],
         'prefer-slash-over-backslash': [RuleStatus.Error, 'always'],
         'type-space-before-paren': [RuleStatus.Error, 'always'],
+        'reject-square-bracket-style': [RuleStatus.Error, 'always'],
```

Comment:
@realmarv to follow same naming style as commitlint (e.g. "type-empty"), let's rename this to `type-with-square-brackets`

Reply:
Done